tests/driver_epd_bw_spi/Makefile

```diff
@@ -0,0 +1,16 @@
+include ../Makefile.tests_common
+
+BOARD := nucleo-f411re
+
+USEMODULE += xtimer
+USEMODULE += epd_bw_spi
+
+CFLAGS += -DDISPLAY_X=200
+CFLAGS += -DDISPLAY_Y=200
```

Comment:
The DISPLAY_X and DISPLAY_Y values should be provided as params of the driver.

Reply:
Do you mean I should add a separate list of defines for the x/y size of various displays?